pyro/infer/mcmc/mh.py

```diff
@@ -0,0 +1,255 @@
+from __future__ import absolute_import, division, print_function
+
+import pyro
+import pyro.distributions as dist
+from pyro import poutine
+from pyro.infer.mcmc.trace_kernel import TraceKernel
+from pyro.util import ng_ones, ng_zeros
+from torch import Tensor as T
+from torch.autograd import Variable as V
+from numpy.random import choice
+from numpy import isfinite
+from torch import log as tlog
+
+_R = '_RETURN'
+
+
+def VT(val):
```

Comment:
nit: This is too clever. Just spell out `Variable(Tensor(...))` everywhere.

Rationale: aliases are nice when you're in a jupyter notebook. But we'll be maintaining this code for a long time, including programming with `grep/sed/awk` while refactoring. It's really difficult to make sweeping changes on a maintained codebase when files use aliases like this. In particular, Neeraj and I will have a lot of work to do when PyTorch eliminates Variables, and each nickname in the code will make our lives more difficult.